unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/CoroutineHelpers/CoroutineStarter.cs

```diff
@@ -5,7 +5,7 @@ public class CoroutineStarter : MonoBehaviour
 {
     private static CoroutineStarter instanceValue;
 
-    private static CoroutineStarter instance
+    public static CoroutineStarter instance
```

Comment:
Is any reason to use the instance instead of the Start/Stop?

Reply:
This is needed for `TaskUtils.RunThrottledCoroutine` as it runs the coroutine on its own to wrap it with an async awaitable, I used the same GameObject to maintain consistency on where the coroutines are being run.